products/sle/main.pm

```diff
@@ -130,6 +130,10 @@ if (check_var('VIRSH_VMM_FAMILY', 'xen') && check_var('VIRSH_VMM_TYPE', 'linux')
     }
 }
 
+unless (check_var('BACKEND', 'svirt') and (check_var('VIRSH_VMM_FAMILY', 'kvm') or check_var('ARCH', 's390x'))) {
+    set_var('BRIDGED_NETWORKING', 1);
+}
```

Comment:
I recommend you replace that by a "is_applicable" function, write the comments from the commit message here explaining why this needs be handled differently and use that "is_applicable" function instead of the variable.